docs/topics/plugin-releases.md

```diff
@@ -41,7 +48,6 @@ Released: July 27, 2021
 * Remote development support
 
 Learn more in:
-* [What's new in Kotlin plugin 2021.2](whatsnew-plugin-20212.md)
 * [What’s New in IntelliJ IDEA 2021.2](https://www.jetbrains.com/idea/whatsnew/)
```

Comment:
Maybe use a permanent link for latest IDEA as well as for other versions? With short link you'll have to update it each new release. 

```suggestion
* [What’s New in IntelliJ IDEA 2021.2](https://www.jetbrains.com/idea/whatsnew/2021-2/)
```